### PR TITLE
Improve the fe-1 documentation

### PR DESCRIPTION
**Pull request by Tyratox**
This PR adds an explanation of the dependency injection pattern used for inter-feature dependencies and adds a description how the public key can be retrieved from the javascript console.

### CODE REVIEW

**Review by 0xdieman — APPROVED** [review]
Nice, some things even became clearer for me after reading that hahaha

File: fe1-web/docs/README.md [hunk 1]
```diff
@@ -344,6 +346,247 @@ which ensures that all constraints defined within the JSON Schema are respected.
...
+
+One may think of the features as being independent packages that are loaded by
+the application. This leads to the question how inter-feature dependencies can
+be handled. For instance there is the lao feature in `src/features/lao` that
+manages the storage of LAO related data such as what the active LAO is. (At the
+time of writing, the application can at a specific point in time only be
+connected to one LAO). If another feature, for example the `evoting` feature,
+needs to know the ID of the active LAO in order to send a message in the correct
+channel, then it is **not** supposed to just import the corresponding function
+directly from the `lao` feature.
```
Comment (pierluca) [inline fe1-web/docs/README.md:361]: if there are no more inter-features dependencies, don't forget to switch depcruise config to treat imports across features as errors.

**Review by pierluca — APPROVED** [review]
pretty clear ! good job !

File: fe1-web/README.md [hunk 2]
```diff
@@ -28,6 +28,9 @@ npm start
 - Opening the project on a phone requires to scan the QR code on the bottom left corner of the page and having [Expo Client app](https://expo.io/tools) installed to display the project. Without the app, a phone may be simulated using the emulator/simulator buttons on the packager
 - Opening the project on a browser requires to click on the *run in web browser* button on the packager
 
+## Retrieving the public key
+In order to retrieve the public key that is used by the frontend, open the javascript console of the web browser of your choice. [This stackoverflow answer](https://webmasters.stackexchange.com/a/77337) gives an overview of how to do so in different browsers if you have never done this. Then look for the log entry `Using the public key: XXX`.
```
[the remark this answers is not in the document]
Comment (Tyratox) [reply fe1-web/README.md:32]: Yes, this is outdated as well 😂 (Was part of the same commit as the redux fix, I think before there were two different messages for when the store was initialized and for when it was rehydrated)

File: fe1-web/docs/README.md (hunk 1 above)
Comment (Tyratox) [reply fe1-web/docs/README.md:361]: 👍

**Pull request merged**